src/components/error/ErrorBoundary.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ReactElement, ReactNode } from 'react';
+import * as Sentry from '@sentry/react';
+import { ApiErrorException } from '@/api/errors';
+import styled from 'styled-components';
+import AlertStripe from 'nav-frontend-alertstriper';
+
+export enum ErrorContext {
+  mainPage = 'mainPage',
+  appRouter = 'appRouter',
+  oversiktContainer = 'oversiktContainer',
+}
+
+interface ContentProps {
+  message: string;
+}
+
+const ErrorContentContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const ErrorRow = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 50em;
+  margin: 2rem;
+`;
+
+const ErrorContent = ({ message }: ContentProps): ReactElement => {
+  return (
+    <ErrorContentContainer>
+      <ErrorRow>
+        <AlertStripe type="feil">{message}</AlertStripe>
```

Comment:
Kan vurdere om vi bør style slik at teksten fyller bredden på alert-stripen (slik det er gjort en del steder i syfomodiaperson https://github.com/navikt/syfomodiaperson/blob/master/src/components/AlertstripeFullbredde.tsx).

Reply:
Jepp, godt innspill, Litt usikker på hvorfor dette ikke er en mulighet i designsystem-komponenten

Reply:
Mener jeg fikk det forklart en gang at man har satt en maksbredde på tekstlinjer i designsystemet (sikkert tilpasset eksterne flater), så derfor ønsker man ikke åpne for det i komponenten.